packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta-serviceapp/create-sign-jwt/index.md

```diff
@@ -12,6 +12,7 @@ For testing purposes, use [this tool](https://www.jsonwebtoken.io/) to generate
 
 The following [JWT claims](/docs/reference/api/oidc/#token-claims-for-client-authentication-with-client-secret-or-private-key-jwt) can be used in the request for a scoped access token:
 
+* `alg` &mdash; The value "RS256" in order for Okta to successfully verify the token using the signing keys provided in the previous step.
```

Comment:
Is RS256 required or the only algorithm supported?

Reply:
@dragos-okta If it's the only algorithm that can be used, I would suggest the following edit:

* `alg` &mdash; The `RS256` asymmetric algorithm is required for Okta to successfully verify the token using the signing keys provided in the previous step.

Reply:
We support 6 algorithms- `RS265`, `RS384`, `RS512`, `ES256`, `ES384`, and `ES512` https://developer.okta.com/docs/guides/build-self-signed-jwt/java/jwt-with-private-key/ and you can use any of them to sign the token.

An important detail to capture here is that the `alg` parameter goes in the _header_ of the jwt, rather than a claim in the payload of the body.

Reply:
@danielzeller-okta Does this update work:

* `alg` -- One of the following supported algorithm values: `RS265`, `RS384`, `RS512`, `ES256`, `ES384`, or `ES512`. The `alg` parameter goes in the header of the JWT rather than a claim in the payload of the body.

Reply:
That, plus "The value must match the algorithm used to sign the jwt"

Reply:
alg -- One of the following supported algorithm values: RS265, RS384, RS512, ES256, ES384, or ES512 is required for Okta to successfully verify the token using the signing keys provided in the previous step. The alg parameter goes in the header of the JWT rather than a claim in the payload of the body.